### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by polylogue2** [cchay3/Kokoro-Beats#1]
Wait we can use github for tasks...

**Comment by cchay3** [comment]
We can?

**Comment by yumetopia** [comment]
wait, how many of us have access to github though? i fear not a lot-

**Comment by cchay3** [comment]
> wait, how many of us have access to github though? i fear not a lot-

No, not a lot. I daresay, only the ones here really have github + a few others.
